### ISSUE TITLE
Consider updates the Curriculum Workshop pages.

### ISSUE THREAD

**Issue by flukeout** [mozilla/mozmaker#48]
After reflecting on the concerns about the amount of information on the workshop page raised by @vazquez and @cassiemc in their review, I thought a way to address that would be to break things up a bit. Instead of having one long landing page that contains the current workshop _and_ all the links to past and previous ones, we could have two separate pages...

* The workshops landing page with the schedule and archives
* The actual workshop page with just the content for that workshop

Here's what it might look like...

![image](https://cloud.githubusercontent.com/assets/25212/13655769/de6d7bc2-e616-11e5-9ae6-b4e02888a32e.png)

**A small drawback**
This has some implications for the CMS however, namely that it will increase the number of templates necessary to run and administer these workshops from one to two, the landing page and the individual workshop page. I guess the landing page doesn't have to be a template per se, but it does need to be easily editable. 

Curious about your thoughts!

**Comment by hannahkane** [comment]
@flukeout - I really like this. Seems like a very elegant solution.

To confirm I understand - the "View recording" links in the Past Workshops section would also go to the individual workshop pages?

I don't think this adds too much complexity. Agreed that the landing page is simple enough in this design to not require a template—will the fancy "Next workshop" section be achievable without a template, though?

**Comment by chadsansing** [comment]
Works for me - does the landing page need any visual elements to break up the text that remains? 

Let me cc @poh, as well, as he'll be working to update materials in his role as producer. @kgorr is also lending a helping hand in the production department, so cc-ing her, as well.

**Comment by KGorr** [comment]
Thanks, @chadsansing. I really like this, and might suggest for our purposes that we never have more than 2 upcoming workshops listed. I think that would make it way to text heavy as Chad suggested. Will there be an opportunity to add images in the descriptions, should the user wish, to help break it up?

**Comment by chadsansing** [comment]
I'm gonna chime in again after more thought and consultations -  having the upcoming/current episode on the landing page is a big part of the inspiration from TTT and making the "work" of the program highly visible and workshoppy, Maybe we should keep it front and center?

**Comment by ldecoursy** [comment]
+1 I'd suggest the main landing page as the place where people participate in the workshop is key, perhaps it's the archives that live on a separate page.

**Comment by flukeout** [comment]
Thanks for the feedback everyone, I'll take another shot at a design that keeps the upcomign / current workshop on the main landing page.

**Comment by flukeout** [comment]
**An update to the original design**
This option uses a tabbed navigation at the top of the page to divide the content into 2 sections.
1. The current / upcoming workshop
2. Workshop description, schedule and archives

Archived workshops would have their own page and would appear similarly to the page on the left (but without the navigation).

The rationale for this design is to emphasize the workshop-related content.

![image](https://cloud.githubusercontent.com/assets/25212/13688418/a3ac37ba-e6d6-11e5-9245-4c05abcfd796.png)

**Comment by hannahkane** [comment]
Nice, @flukeout. I really like this solution—seems to solve the hierarchy problem, while keeping the upcoming workshop front and center.

**Comment by chadsansing** [comment]
I dig the tabs - what say you, @ldecoursy, @kgorr, @amirad, & @poh?

**Comment by amirad** [comment]
Looks good to me!

**Comment by KGorr** [comment]
Yeah! Love it.

**Comment by vazquez** [comment]
Great, I'll begin coding it up and we'll show you a working prototype next.

**Comment by flukeout** [comment]
@vazquez @cassiemc Here's a followup from our critique earlier today where we discussed some changes to the tabs and green box...

![image](https://cloud.githubusercontent.com/assets/25212/13795838/68aedf12-eac0-11e5-8ec8-11a8209e5377.png)

As a "bonus" I messed a bit with the type treatment of the "Join us..." tagline and the workshop title so that they weren't set in the exact same style.

Appreciate your thoughts and input. I'll do another version that also includes the introductory paragraph above the tabs, as discussed.

**Comment by cassiemc** [comment]
Agreed, seems like the intro paragraph should be larger, like a subhead, and the text under the workshop title should just be a regular <p>. I'm not crazy about the italics in the header, I think it's introducing another style when we are doing well to simplify this page.

The tabs seem much better. Would probably call that second tab 'All events'. I was imagining the header 'Workshop Video Stream & Open Agenda' as within the grey box. You could shorten that and take out the word 'workshop'

**Comment by vazquez** [comment]
I'll make these changes in code and get back to you with a screenshot :)
